doc/BUILDING.rst

```diff
@@ -47,7 +47,7 @@ Clone 🐸STT source code (TensorFlow will come as a submdule):
 Bazel: Download & Install
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-First, install Bazel 3.1.0 following the `Bazel installation documentation <https://docs.bazel.build/versions/3.1.0/install.html>`_.
+First, install `Bazelisk <https://docs.bazel.build/versions/main/install-bazelisk.html>`_ or the correct version of `Bazel <https://docs.bazel.build/versions/main/install.html>`_ manually (look inside `STT/tensorflow/.bazelversion <https://github.com/coqui-ai/tensorflow/blob/f8242ebc005f6195b67d58349724e608d4fe45da/.bazelversion>`_). Follow the `Bazelisk installation documentation <https://docs.bazel.build/versions/main/install-bazelisk.html>`_.
```

Comment:
I dont think it's wise to remove the version and refer people to the source tree this way, it's making the doc harder to follow

Reply:
Ideally it would be best to point beginners to bazelisk so they don’t have to track down the correct build of bazel to begin with but I see what you mean.

Reply:
I dont know bazelisk, and usually I prefer to avoid extra layers without specific justification, because the more layers the more risk of bugs, and dealing with bazel tends to be complex by itself already.

Reply:
In this case bazelisk is the official installer for bazel. 

> Bazelisk is a wrapper for Bazel written in Go. It automatically picks a good version of Bazel given your current working directory, downloads it from the official server (if required) and then transparently passes through all command-line arguments to the real Bazel binary. You can call it just like you would call Bazel.

It removes all the issues of locating the correct version.

On arch we couldn't build libstt 1.3.0 with bazel 5.0.0 we had to let bazelisk pick  `.bazelversion` to build it correctly...

Reply:
> In this case bazelisk is the official installer for bazel.
> 
> > Bazelisk is a wrapper for Bazel written in Go. It automatically picks a good version of Bazel given your current working directory, downloads it from the official server (if required) and then transparently passes through all command-line arguments to the real Bazel binary. You can call it just like you would call Bazel.
> 
> It removes all the issues of locating the correct version.
> 
> On arch we couldn't build libstt 1.3.0 with bazel 5.0.0 we had to let bazelisk pick `.bazelversion` to build it correctly...

There were many things in Bazel that were official and yet broken :-). At least dont remove the explicit mention of the version, suggesting `bazelisk` might be a good idea but hiding the version info like that will only add pain IMHO.

Reply:
https://github.com/coqui-ai/STT/blame/main/doc/BUILDING.rst#L12 it's already the way to install bazel 4.2.1 😅